### PR TITLE
Create cd.yml

### PR DESCRIPTION
**Pull request by MolotovHammr**

### CODE REVIEW

**Review by AleksandarGT — DISMISSED** [review]

**Review by BondarencoM — DISMISSED** [review]

**Review by oererenn — DISMISSED** [review]

**Review by michaelbahchevanov — CHANGES_REQUESTED** [review]
There are a few things that should be changed/fixed.

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    branches:
+      - develop
+
+jobs:
+  dockerize:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build image
+      run: docker build . -f Dockerfile -t molotovhammr/kozidataservice:lastest
```
Comment (michaelbahchevanov) [inline .github/workflows/cd.yml]: It spells lastest where it should be latest

Comment (michaelbahchevanov) [inline .github/workflows/cd.yml]:
```suggestion
      run: docker build . -f Dockerfile -t molotovhammr/kozidataservice:latest
```

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    - name: Log into registry
+      run: docker login -u ${{ secrets.DOCKERHUB_USERNAME }} -p ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Push to repository
+      run: docker push molotovhammr/kozidataservice:latest
```
Comment (michaelbahchevanov) [inline .github/workflows/cd.yml:20]: Whereas here it's spelled right.

**Review by larsverp — DISMISSED** [review]
👏🏻

**Comment by BondarencoM** [comment]
@michaelbahchevanov  Do you approve the fix?

**Review by michaelbahchevanov — APPROVED** [review]
That's great! 🚀

**Pull request merged**